test/e2e/storage/dax/dax.go

```diff
@@ -358,3 +360,137 @@ func testDaxOutside(
 	err := e2epod.DeletePodWithWait(f.ClientSet, pod)
 	framework.ExpectNoError(err, "while deleting pod")
 }
+
+// Hugepage page fault testing part starts here
+const (
+	accessHugepagesBinary = "_work/pmem-access-hugepages"
+)
+
+var _ = deploy.DescribeForSome("dax", func(d *deploy.Deployment) bool {
+	// Run these tests for all driver deployments that were created
+	// through the operator, where device mode is Direct.
+	return !d.HasOperator && d.HasDriver && d.Mode == api.DeviceModeDirect
```

Comment:
@okartau: I just noticed that the comment doesn't match the code. Should this run for deployments done with or without the operator? Why that criteria?